vtu11/impl/vtu11_impl.hpp

```diff
@@ -146,6 +150,43 @@ void write( const std::string& filename,
   output.close( );
 }
 
+//ParallelWrite generates a pvtu file and accordingly the vtu pieces in a subfolder
+//Each piece consists of a set of points and cells
+template<typename MeshGenerator, typename Writer = AsciiWriter>
+void parallelWrite( const std::string& path,
+                    std::string baseName,
+                    MeshGenerator& mesh,
+                    const std::vector<DataSet>& pointData,
+                    const std::vector<DataSet>& cellData,
+                    size_t fileId, size_t numberOfFiles,
+                    Writer writer = Writer())
+{
+	//ToDo: Take care of cleaning the folder! not done in this code as Kratos takes care of it
```

Comment:
```suggestion
```
Kratos should not be here
In general the library should not take care of this automatically, this could lead to very unfortunate loss of data
maybe one can add a fct to do this in the future